src/test/java/software/coley/instrument/LiveTest.java

```diff
@@ -52,15 +43,13 @@ public static void setup() {
 	}
 
 	@Test
-	@Timeout(15)
 	@ResourceLock(SERVER) // Use this lock on other tests if they get split later
 	public void test() throws Exception {
 		Process start = null;
 		Client client = null;
 		try {
-			// Start the java-agent on the 'Runner' example application
 			String agent = "-javaagent:" + agentJarPath.toString().replace("\\", "/");
-			ProcessBuilder pb = new ProcessBuilder("C:\\Program Files\\AdoptOpenJDK\\jdk-17\\bin\\java", agent, "-cp", "src/test/resources", "Runner");
+			ProcessBuilder pb = new ProcessBuilder("java", agent, "-cp", "src/test/resources", "Runner");
```

Comment:
oh crap I didn't mean to push my windows path, I was locally testing different versions 🤦